webapp/components/admin_console/ldap_settings.jsx

```diff
@@ -337,7 +337,7 @@ export default class LdapSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.ldap.syncIntervalTitle'
-                            defaultMessage='Synchronization Interval (In Minutes)'
+                            defaultMessage='Synchronization Interval (Minutes)'
```

Comment:
Should we use "Synchronization Interval (Minutes)" or "Synchronization Interval (minutes)"?

Maybe have @lfbrock, @jasonblais help weighing in?